src/ConfigurationRemotingServer/ConfigurationRemotingServer.csproj

```diff
@@ -26,6 +26,10 @@
 
   <Import Project="..\targets\ReferenceEmbeddedCsWinRTProject.targets" />
 
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
+  </ItemGroup>
+

```

Comment:
This one is probably not needed.

Reply:
I believe that Visual Studio added it it to prevent different versions within transient dependencies

Reply:
Ah, then no worry. Thanks for clarification.